src/de/cuuky/varo/configuration/configurations/language/languages/ConfigMessages.java

```diff
@@ -138,6 +138,9 @@ public enum ConfigMessages implements DefaultLanguage {
 	NAMETAG_PREFIX_TEAM("nametag.suffix.noTeam", "%colorcode%%team% &7"),
 	NAMETAG_SUFFIX_TEAM("nametag.suffix.team", "&c %kills%"),
 
+	PLACEHOLDER_NO_TOP_PLAYER("placeholder.noTopPlayer", "Unbekannt"),
+	PLACEHOLDER_NO_TOP_TEAM("placeholder.noTopTeam", "Unbekannt"),

```

Comment:
```suggestion
	PLACEHOLDER_NO_TOP_PLAYER("placeholder.noTopPlayer", "-"),
	PLACEHOLDER_NO_TOP_TEAM("placeholder.noTopTeam", "-"),
```
I think we should keep "-" as the default value